definitions/npm/bull_v1.x.x/flow_>=v0.22.x/bull_v1.x.x.js

```diff
@@ -0,0 +1,56 @@
+declare type $npm$bull$jobType =
```

Comment:
Usually, we just skip the `$npm` an would just write `{packagename}${type|...}`
I opened an issue (#126) for further discussion!